coverage.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# cd to the script directory:
+cd "${0%/*}" || { echo "Couldn't cd to ${0%/*}!"; exit 1; }
+
+export REPORTS_DIR=./build/reports
```

Comment:
Not really for this PR, I guess, but I really don't like how we have the path to the build directory hardcoded everywhere...